core/src/UserSettingsEditor.svelte

```diff
@@ -214,6 +214,22 @@
     }
   }
 
+  function createLinkForStyle(name){
```

Comment:
Please add entries for the editor only via configuration.
https://docs.luigi-project.io/docs/user-settings?section=example

This code would be displayed also in luigi apps which are not using fiori theme.